WebRoot/js/zimbraMail/abook/view/ZmContactSearch.js

```diff
@@ -442,9 +439,11 @@ function(htmlArr, idx, contact, field, colIdx, params) {
 	if (field == ZmItem.F_TYPE) {
 		htmlArr[idx++] = AjxImg.getImageHtml(contact.getIcon());
```

Comment:
Thanks for review input like this.
I missed this as this is not in present functionality.
Good to have it, aw.
https://github.com/Zimbra/zm-web-client/pull/117/commits/97cfa50df7bb0d1b69e5dbf78f80541dc101f0d3